docs/admin/kubernetes-ops/backup-restore.md

```diff
@@ -38,7 +38,7 @@ kubectl get secret cn -n <namespace> -o yaml > secret-backup.yaml
 helm list -n <namespace>
 ```
 
-Keep note of the chart version. For example: `replace-janssen-version`
```

Comment:
this is a reserved keyword `replace-janssen-version` please do not change this anywhere